CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed `category-next/loadCategoryProducts` and `category-next/loadMoreCategoryProducts` module actions to respect `config.entities.optimize` config option.- @hcmlopes (#5070)
 - Use `queryString` library in `task/getUrl` - @cewald (#5263)
 - Changing email & password at once - @Fifciu ([#5315](https://github.com/vuestorefront/vue-storefront/issues/5315))
+- Improved: the code to remove the page key from the query before applying a filter (https://github.com/vuestorefront/vsf-capybara/issues/561)
```

Comment:
Please mark yourself and make link clickable like:
@ymaheshwari1 ([VSF Capybara #561](https://github.com/vuestorefront/vsf-capybara/issues/561))

Reply:
Sure, I will update the changelog